OracleIdentityGovernance/samples/containerizedDB/docker-compose.yaml

```diff
@@ -48,6 +52,11 @@ services:
       - DB_PASSWORD=${DC_ORCL_SYSPWD}
       - DB_SCHEMA_PASSWORD=${DC_RCU_SCHPWD}
       - RCUPREFIX=${DC_RCU_OIMPFX}
+    healthcheck:
+      test: ["CMD", "curl", "-fL", "http://oimadmin:7001/console"]
```

Comment:
If you're allowing the hostname to change, wouldn't this test need to change as well?

Reply:
Actually the PR description was a bit misleading.
Earlier the listen addresses were being set to hard coded values (oimadmin/oimms/soams) in container scripts
These hard coded values have been removed and IPs are used. 
We are not allowing the hostname to change. 
Hardcoded values in container scripts were preventing spinning up of containers using other methods like in k8s where the host values were different from these hard coded values

Reply:
But these are using hostnames. Wouldn't these health checks fail in the same circumstances?

Reply:
This health check is specific to the docker-compose. As in the compose, the hostnames are fixed, it works well

Reply:
Ok, great. Just wanted confirmation.